### ISSUE TITLE
PPOCRV2训练模型报的警告到底什么意思?

### ISSUE THREAD

**Issue by 89080322** [PaddlePaddle/PaddleOCR#5299]
**是检测模型**
中英文超轻量PP-OCRv2模型（13.0M） | ch_PP-OCRv2_xx

下载并用这个训练模型直接训练  

载入配置文件以后 先报错

NING: The pretrained params student2_model.backbone.stage2.1.linear_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.expand_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.expand_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.expand_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.expand_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.expand_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.bottleneck_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.bottleneck_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.bottleneck_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.bottleneck_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.bottleneck_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.linear_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.linear_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.linear_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.linear_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.2.linear_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.expand_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.expand_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.expand_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.expand_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.expand_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.bottleneck_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.bottleneck_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.bottleneck_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.bottleneck_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.bottleneck_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.linear_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.linear_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.linear_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.linear_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.3.linear_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.expand_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.expand_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.expand_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.expand_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.expand_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.bottleneck_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.bottleneck_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.bottleneck_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.bottleneck_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.bottleneck_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.linear_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.linear_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.linear_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.linear_conv.bn._mean not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.4.linear_conv.bn._variance not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.5.expand_conv.conv.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.5.expand_conv.bn.weight not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.5.expand_conv.bn.bias not in model
[2022/01/20 00:13:58] root WARNING: The pretrained params student2_model.backbone.stage2.5.expand_conv.bn._mean not in mode

然后才开始训练  然后训练很久 loss不下来..  测试各种0  .

**Comment by WenmuZhou** [comment]
看起来预训练模型没配置好，方便发一下配置文件吗

**Comment by 89080322** [comment]
Global:
  use_gpu: true
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 2
  save_model_dir: ./output/det_ppv2_cy/inf/
  save_epoch_step: 20
  # evaluation is run every 5000 iterations after the 4000th iteration
  eval_batch_step: [200, 200]
  cal_metric_during_train: False
  pretrained_model: ./output/det_ppv2_cy/best_accuracy
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt

Architecture:
  name: DistillationModel
  algorithm: Distillation
  model_type: det
  Models:
    Student:
      pretrained: ./output/det_ppv2_cy/best_accuracy
      freeze_params: false
      return_all_feats: false
      model_type: det
      algorithm: DB
      Backbone:
        name: MobileNetV3
        scale: 0.5
        model_name: large
        disable_se: True
      Neck:
        name: DBFPN
        out_channels: 96
      Head:
        name: DBHead
        k: 50
    Teacher:
      pretrained: ./output/det_ppv2_cy/best_accuracy
      freeze_params: true
      return_all_feats: false
      model_type: det
      algorithm: DB
      Transform:
      Backbone:
        name: ResNet
        layers: 18
      Neck:
        name: DBFPN
        out_channels: 256
      Head:
        name: DBHead
        k: 50

Loss:
  name: CombinedLoss
  loss_config_list:
  - DistillationDilaDBLoss:
      weight: 1.0
      model_name_pairs:
      - ["Student", "Teacher"]
      key: maps
      balance_loss: true
      main_loss_type: DiceLoss
      alpha: 5
      beta: 10
      ohem_ratio: 3
  - DistillationDBLoss:
      weight: 1.0
      model_name_list: ["Student"]
      name: DBLoss
      balance_loss: true
      main_loss_type: DiceLoss
      alpha: 5
      beta: 10
      ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DistillationDBPostProcess
  model_name: ["Student"]
  key: head_out
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DistillationMetric
  base_metric_name: DetMetric
  main_indicator: hmean
  key: "Student"

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
      - ./train_data/icdar2015/text_localization/train_icdar2015_label.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - CopyPaste:
      - IaaAugment:
          augmenter_args:
            - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': Affine, 'args': { 'rotate': [-10, 10] } }
            - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [960, 960]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 4
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
      - ./train_data/icdar2015/text_localization/test_icdar2015_label.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
#           image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2

**Comment by xiexiaokang** [comment]
解决了吗？

**Comment by northeastsquare** [comment]
./pretrain_models/ResNet50_vd_ssld_pretrained
not 
./pretrain_models/ResNet50_vd_ssld_pretrained.pdparams

**Comment by vedanshthakkar** [comment]
I am facing the same problem using "ResNet50_vd_ssld_pretrained" as a pre-trained model for text detection. Any progress anyone?

**Comment by ericyeyeye** [comment]
I recommend to train PP-OCRv3. When you using Distillation, you have to extract the student model from best_accuracy.
Follow the link and the extract code is in the end.
[https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/knowledge_distillation.md](url)

**Comment by MathewsJosh** [comment]
> I am facing the same problem using "ResNet50_vd_ssld_pretrained" as a pre-trained model for text detection. Any progress anyone?

Did you solved it? how?
